src/minErr.js

```diff
@@ -34,6 +34,7 @@ function minErr(module, ErrorConstructor) {
   ErrorConstructor = ErrorConstructor || Error;
   return function() {
     var SKIP_INDEXES = 2;
+    var MAX_DEPTH = 1;
```

Comment:
I think to make max depth configurable we can,

    var exampleMinErr = minErr('example');
    throw exampleMinErr('one', 'This {0} is {1}', foo, bar);

1. Refactor exampleMinErr to be 

    throw exampleMinErr({code:'one', message: 'This {0} is {1}', messageArgs:[foo, boo]});

and we can keep both styles working.


2.  Include the depth as a special string in the message like  {$maxDepth=3}   
'This {0} is {1} {$maxDepth=3}' so we get the depth
  
 throw exampleMinErr('one', 'This {0} is {1} {$maxDepth=3}', foo, bar);

this Depth will be applied to all the message.

Or Include a depth for each placeholder like 'This {0:d8} is {1:d8} }'

    throw exampleMinErr('one', 'This {0:d8} is {1:d8}', foo, bar);
d8 means that the depth for stringifying this object is 8.

I think the second solution is easier for implementation and have no backward compatibility Issue,  Any Suggestion about that ?